### PR TITLE
Reg 29 creation of route creator page

### PR DESCRIPTION
**Pull request by MarekBrandt**
It is a page for adding markers representing the race route. As for now it is not connected to any other page. I will do this in next task

### CODE REVIEW

File: lib/pages/route_creator.dart [hunk 1]
```diff
@@ -0,0 +1,136 @@
+import 'dart:math';
+
+import 'package:flutter/material.dart';
+import 'package:flutter_map/flutter_map.dart';
+import 'package:latlong2/latlong.dart';
+
+import '../widgets/app_header.dart';
+import '../widgets/complex_marker.dart';
```
Comment (bartoz1) [inline lib/pages/route_creator.dart:8]: I'm not a big fan of relative paths and I would consider using package paths (hope that's the right name 😅)

File: lib/pages/route_creator.dart [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+                    urlTemplate:
+                        'https://tile.openstreetmap.org/{z}/{x}/{y}.png',
+                    userAgentPackageName: 'com.regattaBuddy.app',
+                  ),
+                  PolylineLayer(
+                    polylines: [
+                      Polyline(
+                          points: markers
+                              .map(
+                                  (complexMarker) => complexMarker.marker.point)
```
Comment (bartoz1) [inline lib/pages/route_creator.dart:94]: The formatting looks weird

**Review by bartoz1 — APPROVED** [review]
Would be much easier with some screenshots

File: lib/pages/route_creator.dart (hunk 1 above)
Comment (MarekBrandt) [reply lib/pages/route_creator.dart:8]: I'll fix both in next pr. Right now I merge

**Pull request merged**